Code/Framework/AzCore/Tests/Math/SimdMathTests.cpp

```diff
@@ -191,7 +261,71 @@ namespace UnitTest
 
         for (int32_t i = 0; i < VectorType::ElementCount; ++i)
         {
-            EXPECT_THAT(testStoreValues[i], 1);
+            EXPECT_THAT(testStoreValues[i], ::testing::Eq(1));
+        }
+    }
+
+    template<typename VectorType>
+    void TestSplatFirst()
+    {
+        float testLoadValues[4] = { 3.5f, 0.0f, 0.0f, 0.0f };
```

Comment:
Not a big deal either way, but I guess you could also save some repetition here by writing a TestSplatNth which takes a parameter from 0 to 3 and sets that element to 3.5f and all the others to 0.0f.

Reply:
I tried to follow the same patter as other tests, keep them simple and separate.